### ISSUE TITLE
Unable to load schema from 'https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json': unexpected end of file.(768)

### ISSUE THREAD

**Issue by adriancuadrado** [microsoft/vscode-azurearmtools#1443]
VSCode reports that error when I create a parameters file. For now, as a workaround, I had to change to this schema instead:
```
https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json
```

**Comment by StephenWeatherford** [comment]
Hmmm, second report of this (see https://github.com/microsoft/vscode-azurearmtools/issues/1441).  Asking around.

@adriancuadrado  Does it actually cause any functionality problems?  Thx.

**Comment by adriancuadrado** [comment]
@StephenWeatherford I can't use the VSCode autocomplete mechanism to write the contents of the ARM Template.

**Comment by StephenWeatherford** [comment]
Turns out it's a vscode in version 1.63.0: https://github.com/microsoft/vscode/issues/140227.  It's been fixed in the Insiders version 1.64, if you don't mind installing vscode insiders (https://code.visualstudio.com/insiders/)
